pkg/dump/dump.go

```diff
@@ -148,15 +155,16 @@ func getConsumerGroupsConfiguration(ctx context.Context, group *errgroup.Group,
 			getConsumerGroupsFunc = GetAllConsumerGroupsWithoutConsumers
 		}
 
-		consumerGroups, err = getConsumerGroupsFunc(ctx, client, config.SelectorTags)
+		consumerGroups, err = getConsumerGroupsFunc(ctx, client, config.SelectorTags, SelectTag)
```

Comment:
Can we add a comment here explaining the usage of the select tag and what are its effects?